cluster-state-service/handler/api/v1/instanceapis.go

```diff
@@ -122,7 +125,7 @@ func (instanceAPIs ContainerInstanceAPIs) ListInstances(w http.ResponseWriter, r
 		}
 	}
 
-	var instances []types.ContainerInstance
+	var instances []storetypes.ContainerInstanceWrapper
```

Comment:
Good idea, I like those names better. I've changed them to `VersionedContainerInstance` and `VersionedTask`.